### PR TITLE
Testing | Packaging v2

### PR DESCRIPTION
**Pull request by alexAubin**

### CODE REVIEW

**Comment by alexAubin** [comment]
!testme

File: scripts/change_url [hunk 1]
```diff
@@ -80,29 +23,7 @@ ynh_systemd_action --service_name=$app --action="stop" --log_path="/var/log/$app
 #=================================================
 ynh_script_progression --message="Updating NGINX web server configuration..."
 
-nginx_conf_path=/etc/nginx/conf.d/$old_domain.d/$app.conf
```
Comment (Salamandar) [inline scripts/change_url:83]: Line 19, `--line_match="Proxy listening"` should be remove because action=stop

File: scripts/upgrade [hunk 2]
```diff
@@ -132,13 +55,6 @@ ynh_script_progression --message="Starting a systemd service..."
 
 ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log" --line_match="Proxy listening"
 
-#=================================================
```
Comment (Salamandar) [inline scripts/upgrade:135]: Here, ynh_systemd_action start fails because it’s already started. You should stop it at the beginning of the upgrade.

**Review by Salamandar — CHANGES_REQUESTED** [review]

**Comment by Salamandar** [comment]
!testme

**Comment by Salamandar** [comment]
@alexAubin fixed now !

**Comment by alexAubin** [comment]
Cheers :O

**Pull request merged**